public/data/sumitmukharjeeeeee.json

```diff
@@ -0,0 +1,15 @@
+{
+    "name": "Sumit Mukharjee",
+    "bio": "Learning and Exploring",
+    "avatar": "https://github.com/sumitmukharjeeeeee.png",
+    "links": [{
```

Comment:
Please move the brace `{` to the next line, and format it please @sumitmukharjeeeeee